RLBotFramework/utils/class_importer.py

```diff
@@ -1,75 +1,73 @@
 import importlib
 import inspect
-
 import os
+import sys
+import uuid
 
 from RLBotFramework.agents.base_agent import BaseAgent
-from RLBotFramework.utils.logging_utils import log_warn
 
 
-def is_extends_base_class(module, base_class):
-    super_classes = module.__bases__
-    for super_class in super_classes:
-        if super_class == base_class:
-            return True
-    return False
+class AgentLoadData:
+    def __init__(self, module_spec, base_class):
+        self.module_spec = module_spec
+        self.base_class = base_class
+        self.agent_class = None

```

Comment:
I think there should be a method to get the agent class instead of directly accessing the agent class
In case we want to perform some safe logic around the agent_class being None

Reply:
Yep, could be useful. Added it, but didn't do any logic for None right now because currently it is set in the constructor during reload().